upload/admin/controller/extension/shipping/mds.php

```diff
@@ -157,7 +157,8 @@ public function index() {
         $data['shipping_mds_is_auto_create_waybill'] = $this->config->get('shipping_mds_is_auto_create_waybill');
         $data['shipping_mds_is_auto_create_address'] = $this->config->get('shipping_mds_is_auto_create_address');
 
-        $data['default_collivery_from_addresses'] = array();
+
+        $data['default_collivery_from_addresses'] = $this->collivery->getAddresses(array());
```

Comment:
`Collivery::getAddresses()` has a `private` scope. You can't call it outside of the class.
Please update the `getAddresses` method to be `public`

Also no need to pass in the `array()` the default is set in `getAddresses(array $filter = [])`